dvc/repo/plots/__init__.py

```diff
@@ -15,6 +18,11 @@ def diff(self, *args, **kwargs):
     def modify(self, path, props=None, unset=None):
         from dvc.dvcfile import Dvcfile
 
+        props = props or {}
+        template = props.get("template")
+        if template:
+            _evaluate_templatepath(self.repo, template)
```

Comment:
```suggestion
            self.repo.plot_templates.get_template(template)
```
If #3994 gets merged first